nimquery.nim

```diff
@@ -912,7 +912,7 @@ proc parseHtmlQuery*(queryString: string,
                 else: doAssert(false) # can't happen
 
             of CombinatorKinds:
-                parts.add initQueryPart(demands, lexer.current.kind.ord.Combinator)
+                parts.add initQueryPart(demands, lexer.current.kind.Combinator)
```

Comment:
This change is why the devel build is failing - Nim won't allow direct enum to enum conversions in the future. Revert this and it should work